### PR TITLE
Added the OIDC flow shared bits, and duration

### PR DESCRIPTION
**Pull request by tdykstra**
Copies https://github.com/dotnet/docs/pull/40729

Contributes to #32479

### CODE REVIEW

File: .github/workflows/quest-bulk.yml [hunk 1]
```diff
@@ -25,25 +29,20 @@ jobs:
...
         id: bulk-sequester
         uses: dotnet/docs-tools/actions/sequester@main
         env:
           ImportOptions__ApiKeys__GitHubToken: ${{ secrets.GITHUB_TOKEN }}
           ImportOptions__ApiKeys__QuestKey: ${{ secrets.QUEST_KEY }}
-          ImportOptions__ApiKeys__AzureAccessToken: ${{ env.AZURE_ACCESS_TOKEN }}
+          ImportOptions__ApiKeys__AzureAccessToken: ${{ steps.azure-oidc-auth.outputs.access-token }}
           ImportOptions__ApiKeys__SequesterPrivateKey: ${{ secrets.SEQUESTER_PRIVATEKEY }}
           ImportOptions__ApiKeys__SequesterAppID: ${{ secrets.SEQUESTER_APPID }}
         with:
```
Comment (BillWagner) [inline .github/workflows/quest-bulk.yml:48]: There was a mistake in the source docs PR that we fixed yesterday. You need to add this line after line 52:

```
duration: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.duration || 5 }}
```

See https://github.com/dotnet/docs/blob/main/.github/workflows/quest-bulk.yml#L51C11-L51C105

File: .github/workflows/quest.yml [hunk 2]
```diff
@@ -51,7 +52,7 @@ jobs:
         uses: dotnet/docs-tools/actions/sequester@main
         env:
           ImportOptions__ApiKeys__GitHubToken: ${{ secrets.GITHUB_TOKEN }}
-          ImportOptions__ApiKeys__AzureAccessToken: ${{ env.AZURE_ACCESS_TOKEN }}
+          ImportOptions__ApiKeys__AzureAccessToken: ${{ steps.azure-oidc-auth.outputs.access-token }}
           ImportOptions__ApiKeys__QuestKey: ${{ secrets.QUEST_KEY }}
           ImportOptions__ApiKeys__SequesterPrivateKey: ${{ secrets.SEQUESTER_PRIVATEKEY }}
           ImportOptions__ApiKeys__SequesterAppID: ${{ secrets.SEQUESTER_APPID }}
```
Comment (BillWagner) [inline .github/workflows/quest.yml:58]: Same comment here.  Need to add 

```
duration: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.duration || 5 }}
```
As in https://github.com/dotnet/docs/blob/main/.github/workflows/quest-bulk.yml#L51C11-L51C105

**Review by BillWagner — APPROVED** [review]
We found a mistake while testing. I added comments with the two additions needed. Then, this is ready to ship.

File: .github/workflows/quest.yml (hunk 2 above)
Comment (tdykstra) [reply .github/workflows/quest.yml:58]: In dotnet/docs, this line is only in Quest-bulk.yml,  not Quest.yml -- should it only be in Quest-bulk.yml?

**Pull request merged**